sdk/modelsrepository/Azure.Iot.ModelsRepository/samples/ModelsRepositoryClientSamples/ModelResolutionSamples.cs

```diff
@@ -17,26 +17,37 @@ internal class ModelResolutionSamples
 
         public static void ClientInitialization()
         {
-            // When no Uri is provided for instantiation. The global Azure IoT Models Repository endpoint is used
-            // and the dependency model resolution option is set to TryFromExpanded.
-            var client = new ModelsRepositoryClient();
+            #region Snippet:IoTModelsRepositorySamplesCreateServiceClientSimpleWithGlobalEndpoint
```

Comment:
nit: I think you can simplify the names of the snippets a bit so they don't read too long.
can be simplified to something like : `ModelsRepositorySampleCreateServiceClientWithGlobalEndpoint`. 
`IoT` and `Simple` can be removed I think ...

Reply:
Mmm yea good call

Reply:
Done.